src/styles/stories/Margins/MarginExample.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { FC } from 'react';
```

Comment:
Lets use `import type` [TS feature](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html) 
```suggestion
import type { FC } from 'react';
```